.github/workflows/master_deploy-backend.yml

```diff
@@ -22,13 +22,13 @@ jobs:
         with:
           python-version: "3.8"
 
-      - name: Create and start virtual environment
-        run: |
-          python -m venv venv
-          source venv/bin/activate
+      #- name: Create and start virtual environment
+      #  run: |
+      #    python -m venv venv
+      #    source venv/bin/activate
 
-      - name: Install dependencies
-        run: pip install -r backend/requirements.txt
+      #- name: Install dependencies
+      #  run: pip install -r backend/requirements.txt
```

Comment:
A question here: it seems like this was already not required before because requirements are being installed by `start_backend.sh` and that's why you removed it, correct?

Reply:
If I understand correctly, this step installs the python dependencies within the github action runner. But the github action runner only copies the code to Azure (without the venv), so installing the deps here doesn't do anything I think.

Installing the deps here also probably wouldn't work because the GH actions runner is not guaranteed to run a compatible system (architecture / base OS) as the Azure server so the venv has to be recreated on the azure side anyways.